app/models/participant.rb

```diff
@@ -13,4 +13,10 @@ class Participant < ApplicationRecord
   def position_name
     position&.short_name || 'None'
   end
+
+  def self.search(params)
+    participants = Participant.all
```

Comment:
Do you need to get `all` in order to search? Can't we just query for the exactly searched terms and not fetch all and then filter?

Reply:
You are right @SenegeacStefan, it's a wip ticket, thanks for pointing out this. :+1: